### DISCUSSION TITLE
prevent markdown url hiding after moving cursor to different line

### DISCUSSION THREAD

**Discussion by davidgao7** [LazyVim/LazyVim#2560] [Q&A] [answered]
Hello everyone, is there any way to disable auto hiding the url in markdown?

The url only show when the cursor is on the line
<img width="570" alt="Screenshot 2024-02-15 at 10 59 57 AM" src="https://github.com/LazyVim/LazyVim/assets/39502459/d74f17ff-c984-4209-aaf5-cbbe37a097a4">
<img width="705" alt="Screenshot 2024-02-15 at 10 59 41 AM" src="https://github.com/LazyVim/LazyVim/assets/39502459/6554a469-d3d6-44f6-bdd3-624eefb45943">

**Comment by dpetka2001** [comment]
`:h conceallevel`

**Reply by dpetka2001** [reply, marked as answer]
You can also press `<leader>uc` to toggle off the `conceallevel` and always show everything or set the option to whatever you like in your `options.lua` file.
